controllers/snapshot/snapshot_adapter.go

```diff
@@ -217,8 +219,18 @@ func (a *Adapter) EnsureStaticIntegrationPipelineRunsExist() (controller.Operati
 			}
 		}()
 
+		var errsForPLRCreation error
 		for _, integrationTestScenario := range *integrationTestScenarios {
 			integrationTestScenario := integrationTestScenario //G601
+			scenarioStatusCondition := meta.FindStatusCondition(integrationTestScenario.Status.Conditions, gitops.IntegrationTestScenarioValid)
+			if scenarioStatusCondition == nil || scenarioStatusCondition.Status == metav1.ConditionFalse {
```

Comment:
I'm thinking we may want to be less strict (at least for starters) and not have the `scenarioStatusCondition == nil` condition here.